adserver/tests/test_advertiser_dashboard.py

```diff
@@ -575,6 +588,19 @@ def test_flight_request_view(self):
         self.assertTrue(resp["location"].startswith("/accounts/login/"))
 
         # Regular user - access to this advertiser
+        self.client.force_login(self.user)
+
+        # Make it a reporter who can't access
+        member = UserAdvertiserMember.objects.get(user=self.user, advertiser=self.advertiser)
+        member.role = "Reporter"
```

Comment:
I think we probably want constants for these? Not as important in tests, of course, but feels weird to have so many strings scattered around.

Reply:
Sure. There are actually constants and I should use them in the tests.